packages/router/router.ts

```diff
@@ -1799,7 +2120,7 @@ async function callLoaderOrAction(
 
     result = await handler({
       params: match.params,
-      request: createRequest(location, submission),
+      request,
```

Comment:
This is the old request creation spot - way down at the point we call a `loader`/`action`